tests/localosmosis/scripts/setup.sh

```diff
@@ -55,8 +55,8 @@ edit_genesis () {
     dasel put string -f $GENESIS '.app_state.mint.params.mint_denom' "uosmo"
     dasel put string -f $GENESIS '.app_state.mint.params.epoch_identifier' "day"
 
-    # Update gamm module
-    dasel put string -f $GENESIS '.app_state.gamm.params.pool_creation_fee.[0].denom' "uosmo"
+    # Update swaprouter module
+    dasel put string -f $GENESIS '.app_state.swaprouter.params.pool_creation_fee.[0].denom' "uosmo"
```

Comment:
```suggestion
    # Update swaprouter module
    dasel put string -f $GENESIS '.app_state.swaprouter.params.pool_creation_fee.[0].denom' "uosmo"
    dasel put string -f $GENESIS '.app_state.gamm.params.pool_creation_fee.[0].denom' "uosmo"
```

Reply:
I could be not tracking something, but isn't the param for pool creation fee under gamm getting removed in place of swaprouter? @p0mvn

Reply:
It's still present only for migration purposes during the upgrade. We will remove it completely in the upgrade after.

AFAIU, changing it in gamm shouldn't make a difference

Reply:
Right, I guess my point is, if we make this requested change, then when we do remove the param from gamm in the next upgrade, if we forget to remove it from here as well then localosmosis will fail to start. Basically no up side and only potential to make a mistake in the future.

Reply:
It helps if your using multi-version builds now, no? (e.g. accidentally have this file, or this produced genesis in v13)